docs/_advanced/middleware_global.md

```diff
@@ -6,7 +6,7 @@ order: 5
 ---
 
 <div class="section-content">
-Global middleware is run for all incoming events before any listener middleware. You can add any number of global middleware to your app by utilizing `app.use(fn)`. The middleware function `fn` is called with the same arguments as listeners and an additional `next` function.
+Global middleware is run for all incoming requests before any listener middleware. You can add any number of global middleware to your app by utilizing `app.use(fn)`. The middleware function `fn` is called with the same arguments as listeners and an additional `next` function.
```

Comment:
I can't comment in the exact location, but can you also update the references to `event` on lines 19 and 32 in this file, please?

Reply:
I have made the changes you requested. Please take a look at it and let me know if this needs any changes.
Thanks